atmos.yaml

```diff
@@ -128,3 +128,53 @@ commands:
         steps:
           - echo Playing ping-pong...
           - echo pong
+
+# Integrations
+integrations:
+
+  # Atlantis
+  # https://www.runatlantis.io/docs/repo-level-atlantis-yaml.html
+  atlantis:
+    path: "atlantis.yaml"
```

Comment:
Could this also be generated to stdout ? It's possible that there are other atlantis configurations required outside of the `projects` key which is what this portion is generating.

Reply:
or is this `path` also read in to find and update the `projects` key in-place? If so, that's good enough too.

Reply:
@nitrocode you could specify `/dev/stdout` to write to stdout.

Reply:
(we should document that @aknysh )